pkgs/servers/mlflow-server/default.nix

```
@@ -0,0 +1,34 @@
+{lib, python3Packages, writeText}:
```

for applications, you should just import the interpreter, then access the package set through the pkgs attr
```suggestion
{lib, python, writeText}:
```
see other comments for setting python version

also, you can use an alias to not repeat yourself so much
```suggestion
{lib, python, writeText}:

let
  py = python.pkgs
in
```